apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/GraphQLCompiler.kt

```diff
@@ -53,6 +53,14 @@ class GraphQLCompiler {
       val transformedQueryOutput = TransformedQueryOutput(args.packageNameProvider)
       transformedQueryOutput.apply { visit(ir) }.writeTo(transformedQueriesOutputDir)
     }
+
+    args.operationOutputDir?.let { operationIdOutputDir ->
```

Comment:
Nitpicking: `operationIdOutputDir` could be renamed to `operationOutputDir` so that it's consistent with other use cases.